### DISCUSSION TITLE
Damage SMC?

### DISCUSSION THREAD

**Discussion by Bambus89** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#180] [Q&A] [answered]
Is it possible, that AlDente could damage the SMC especially in M1 Macs in any way?
There is no officially way told from apple to reset SMC on M1. The "Shutdown and Close Lid at least for 30sec method" is not officially told by apple, right?

So is it possible, that AlDente could damage SMC in a way that you have to send the Macbook to apple?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @Bambus89,
hundreds of people have AlDente working on M1 MacBooks without any issue. Therefore, I don't think AlDente will damage your SMC.

Matthias
